utils/mock-data-loader/bss/deals/deals-to-submit-to-tfm.js

```diff
@@ -0,0 +1,242 @@
+    'supplier-correspondence-address-line-2': '',
+    'supplier-correspondence-address-line-3': '',
+    'supplier-correspondence-address-town': '',
+    'supplier-correspondence-address-postcode': '',
+    'industry-sector': {
+      code: '1005',
+      name: 'Construction',
+    },
+    'industry-class': {
+      code: '42910',
+      name: 'Construction of water projects',
+    },
+    'sme-type': 'Medium',
+    'supply-contract-description': 'TEST',
+    legallyDistinct: 'false',
+    'indemnifier-companies-house-registration-number': '',
+    'indemnifier-name': '',
+    'indemnifier-address-country': {},
+    'indemnifier-address-line-1': '',
+    'indemnifier-address-line-2': '',
+    'indemnifier-address-line-3': '',
+    'indemnifier-address-town': '',
+    'indemnifier-address-postcode': '',
+    'indemnifier-correspondence-address-country': {},
+    'indemnifier-correspondence-address-line-1': '',
+    'indemnifier-correspondence-address-line-2': '',
+    'indemnifier-correspondence-address-line-3': '',
+    'indemnifier-correspondence-address-town': '',
+    'indemnifier-correspondence-address-postcode': '',
+    indemnifierCorrespondenceAddressDifferent: '',
+    'buyer-name': 'A1 Test',
+    'buyer-address-country': {
+      name: 'United Kingdom',
+      code: 'GBR',
+    },
+    'buyer-address-line-1': 'TEST',
+    'buyer-address-line-2': 'TEST',
+    'buyer-address-line-3': 'TEST',
+    'buyer-address-town': 'TEST',
+    'buyer-address-postcode': 'TEST',
+    destinationOfGoodsAndServices: {
+      name: 'United Arab Emirates',
+      code: 'ARE',
+    },
+    supplyContractValue: '5000000.00',
+    supplyContractCurrency: {
+      text: 'GBP - UK Sterling',
+      id: 'GBP',
+      currencyId: 12,
+    },
+    supplyContractConversionRateToGBP: '',
+    'supplyContractConversionDate-day': '',
+    'supplyContractConversionDate-month': '',
+    'supplyContractConversionDate-year': '',
+  },
+  summary: {},
+  comments: [],
+  editedBy: [
+    {
+      date: '1626968578071',
+      username: 'maker1@ukexportfinance.gov.uk',
+      roles: [MAKER],
+      bank: {
+        id: '9',
+        name: 'UKEF test bank (Delegated)',
+        emails: ['checker@ukexportfinance.gov.uk'],
+      },
+      userId: '60f7d72654f99900074c0a6d',
+    },
+    {
+      date: '1626971072253',
+      username: 'maker1@ukexportfinance.gov.uk',
+      roles: [MAKER],
+      bank: {
+        id: '9',
+        name: 'UKEF test bank (Delegated)',
+        emails: ['checker@ukexportfinance.gov.uk'],
+      },
+      userId: '60f7d72654f99900074c0a6d',
+    },
+  ],
+  mandatoryCriteria,
+  supportingInformation: {
+    validationErrors: {
+      count: 0,
+      errorList: {
+        exporterQuestionnaire: {},
+      },
+    },
+    securityDetails: {
+      exporter: 'Test',
+    },
+  },
+  exporter: {
+    companyName: 'Auto Test 1',
+  },
+};
+
+const deals = new Array(25).fill({}).map((_item, index) => ({
```

Comment:
Does filling with all the same data work? A quick look would suggest yes, but it would be good to double check (ie we always select by _id and not by name or anything else thats silly in tests for instance?)

Is there any easy way we could vary some of this data? How does it display on the table, and can we vary that at all? Even if it is `nameOfDeal01` `nameOfDeal02` etc

Reply:
The default values are
```js
const dealDefaults = {
  dealType: 'BSS/EWCS',
  submissionType: 'Automatic Inclusion Notice',
  updatedAt: Date.now(),
  bank: {...},
  details: {
    status: 'Draft',
    maker: {...},
    created: nowTimestamp,
  },
  eligibility: {...},
  submissionDetails: {...},
  summary: {},
  comments: [],
  editedBy: [...],
  mandatoryCriteria,
  supportingInformation: {...},
  exporter: {...}
}
```
The only field that needs care is `submission details`
```js
  submissionDetails: {
    status: 'Incomplete',
    'supplier-type': 'Exporter',
    'supplier-companies-house-registration-number': 'TEST',
    'supplier-name': 'Auto Test 1',
    'supplier-address-country': {
      name: 'Australia',
      code: 'AUS',
    },
    'supplier-address-line-1': 'TEST',
    'supplier-address-line-2': 'TEST',
    'supplier-address-line-3': 'TEST',
    'supplier-address-town': 'TEST',
    'supplier-address-postcode': 'TEST',
    'supplier-correspondence-address-is-different': 'false',
    'supplier-correspondence-address-country': {},
    'supplier-correspondence-address-line-1': '',
    'supplier-correspondence-address-line-2': '',
    'supplier-correspondence-address-line-3': '',
    'supplier-correspondence-address-town': '',
    'supplier-correspondence-address-postcode': '',
    'industry-sector': {
      code: '1005',
      name: 'Construction',
    },
    'industry-class': {
      code: '42910',
      name: 'Construction of water projects',
    },
    'sme-type': 'Medium',
    'supply-contract-description': 'TEST',
    legallyDistinct: 'false',
    'indemnifier-companies-house-registration-number': '',
    'indemnifier-name': '',
    'indemnifier-address-country': {},
    'indemnifier-address-line-1': '',
    'indemnifier-address-line-2': '',
    'indemnifier-address-line-3': '',
    'indemnifier-address-town': '',
    'indemnifier-address-postcode': '',
    'indemnifier-correspondence-address-country': {},
    'indemnifier-correspondence-address-line-1': '',
    'indemnifier-correspondence-address-line-2': '',
    'indemnifier-correspondence-address-line-3': '',
    'indemnifier-correspondence-address-town': '',
    'indemnifier-correspondence-address-postcode': '',
    indemnifierCorrespondenceAddressDifferent: '',
    'buyer-name': 'A1 Test',
    'buyer-address-country': {
      name: 'United Kingdom',
      code: 'GBR',
    },
    'buyer-address-line-1': 'TEST',
    'buyer-address-line-2': 'TEST',
    'buyer-address-line-3': 'TEST',
    'buyer-address-town': 'TEST',
    'buyer-address-postcode': 'TEST',
    destinationOfGoodsAndServices: {
      name: 'United Arab Emirates',
      code: 'ARE',
    },
    supplyContractValue: '5000000.00',
    supplyContractCurrency: {
      text: 'GBP - UK Sterling',
      id: 'GBP',
      currencyId: 12,
    },
    supplyContractConversionRateToGBP: '',
    'supplyContractConversionDate-day': '',
    'supplyContractConversionDate-month': '',
    'supplyContractConversionDate-year': '',
  },
```
As far as I can tell, all of these could reasonably be duplicated, regardless I'll vary some values so they're distinguishable